.github/workflows/version.yml

```
@@ -0,0 +1,115 @@
+            # Get commits since last tag
+            COMMITS=$(git log $LAST_TAG..HEAD --format="%s")
+          fi
+          echo "commits<<EOF" >> $GITHUB_OUTPUT
+          echo "$COMMITS" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+      - name: Determine version bump
+        id: bump
+        run: |
+          COMMITS="${{ steps.get_commits.outputs.commits }}"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)